### PR TITLE
#1138 support `Long` columns in EqualsClause filter method

### PR DESCRIPTION
**Pull request by junaidzm13**
- this change also adds tests for EqualsClause filter method to make sure that this bug doesn't get re-introduced.

### CODE REVIEW

**Comment by naleeha** [comment]
image
I think the test results looks really good, very descriptive & we get test for each case

File: vuu/src/test/scala/org/finos/vuu/core/filter/FilterClauseTest.scala [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      ("Int", "size", "7", false),
+      ("Long", "timestamp", "5", true),
+      ("Long", "timestamp", "0", false),
+      ("Float", "priceInFloat", "100.0", true),
+      ("Float", "priceInFloat", "100.1", false),
+      ("Double", "priceInDouble", "100.0", true),
+      ("Double", "priceInDouble", "100.1", false),
+      ("Boolean", "true-col", "true", true),
+      ("Boolean", "true-col", "false", false),
+      ("Null", "null-col", "null", false)
```
Comment (naleeha) [inline vuu/src/test/scala/org/finos/vuu/core/filter/FilterClauseTest.scala:37]: I think there is two more edge cases here
1. if the there is a column with value null and we call filter(null) i would expect it to return true.
2. If there is a column that is no in the test data and you try to filter for that column i guess we need to report that as error.. not sure what it does currently 

The second one i would write its own test as its edge cases

**Review by naleeha — APPROVED** [review]
